### PR TITLE
http only cookie handling

### PR DESCRIPTION
**Pull request by pozylon**
🤟 enables authenticated queries in apollo playground after using login methods

### CODE REVIEW

File: storefront/.env.defaults [hunk 1]
```diff
@@ -5,3 +5,4 @@ SKIP_INVALID_REMOTES=false
 UNCHAINED_ENDPOINT=http://localhost:4010/graphql
 GETCOCKPIT_ENDPOINT=http://localhost:4020
 GETCOCKPIT_TOKEN=token
+COOKIE_DOMAIN=
```
Comment (schmidsi) [inline storefront/.env.defaults:8]: Shouldn't it be `localhost:3000`?

File: storefront/api/remotes/setLoginCookie.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        } = loginMethodResponse;
+        if (!tokenExpires || !token) {
+          throw new Error(
+            `Query token & tokenExpires on ${loginMethodResponse} so we can set the cookie server-side for you`,
+          );
+        }
+        const formattedExpires = new Date(tokenExpires).toUTCString();
+        responseContext.res.setHeader(
+          'Set-Cookie',
+          `meteor_login_token=${token}; Path=/; HttpOnly; Expires=${formattedExpires}; ${cookieDomain}`,
```
Comment (schmidsi) [inline storefront/api/remotes/setLoginCookie.ts:68]: Do we still need `meteor_login_token` can't we use `unchained_login_token`?

Comment (pozylon) [reply storefront/api/remotes/setLoginCookie.ts:68]: Yeah, good idea. Maybe even a much more general name like "access_token"? That'd be here: https://github.com/unchainedshop/unchained/blob/a842e5d1321cf6e1319b95f4d42a22dec025a226/packages/api/user-context.js#L7

File: storefront/.env.defaults (hunk 1 above)
Comment (pozylon) [reply storefront/.env.defaults:8]: It will work when left empty and set to localhost automatically by the browser

**Pull request merged**